contrib/convert_twitter_archive_to_json_data_source.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+"""This file converts your Twitter Archive data file into a simple JSON file like this:
+
+{
+    "2019-01-01": 123,
+    "2019-01-02": 456,
+    ...
+}
+
+The resulting file serves as an input file to the GitHubPoster JSON data source.
+
+More on your Twitter Archive: https://help.twitter.com/en/managing-your-account/how-to-download-your-twitter-archive
+
+More on GitHubPoster project: https://github.com/wzyboy/GitHubPoster
+"""
+
+import json
+import argparse
+from datetime import datetime
+from collections import Counter
+
+
+# Wed Jul 08 15:04:01 +0000 2009
+TIMESTAMP_FMT = '%a %b %d %H:%M:%S %z %Y'
+
+
+def convert(input_file, output_file):
+
+    with open(input_file, 'r') as f:
+        content = f.read()
+
+    # Read JSON payload from the JavaScript file
+    preamble = 'window.YTD.tweet.part0 = '
+    content = content[len(preamble):]
+
+    counter = Counter()
+    tweets = json.loads(content)
+    for tweet in tweets:
+        date = str(datetime.strptime(tweet['tweet']['created_at'], TIMESTAMP_FMT).date())
```

Comment:
use `pendulum` like other parts?

Reply:
I tried to use `pendumlum` but found it unecessary complex (e.g. it does not use standard `strftime()` tokens). I'm personally more inclined to use stdlib to parse date and time.

Reply:
> I tried to use `pendumlum` but found it unnecessary complex (e.g. it does not use standard `strftime()` tokens). I'm personally more inclined to use stdlib to parse date and time.

Yes agree with you but for this repo `pendumlum` is handy for many loader so...I choosed it.